translations/translationtool/src/translationtool.php

```diff
@@ -561,18 +628,68 @@ private function rrmdir($path) {
 
 		rmdir($path);
 	}
+
+	public function log(string $message) {
+		if ($this->verbose == 0) {
+			return;
+		}
+		echo " > " . $message . PHP_EOL;
+	}
+}
+
+// arguments handle
+$task = '';
+$usage = false;
+$verbose = 0;
+$returnValue = true;
+
+$index = 0;
+foreach ($argv as $arg) {
+	$index++;
+	if ($index == 1) {
+		$TOOLNAME = $arg;
+		continue;
+	}
+	switch($arg) {
+		case '-h':
+		case '--help':
+			$usage = true;
+			break;
+		case '-v':
+		case '--verbose':
+			$verbose++;
+			break;
+		case 'create-pot-files':
+		case 'convert-po-files':
+		case 'check-files':
+			$task = $arg;
+			break;
+		default:
+			echo "Unknown command parameter : " . $arg . PHP_EOL;
+			$usage = true;
+			$returnValue = false;
+			break;
+		}
 }
 
 // read the command line arguments
-if(count($argv) < 2) {
+if(empty($task) && !$usage) {
 	echo 'Missing arguments' . PHP_EOL;
-	echo 'call "' . $argv[0] . ' $task [$appName]' . PHP_EOL;
-	echo '$task: create-pot-files || convert-po-files' . PHP_EOL;
-	return false;
+	$usage = true;
+	$returnValue = false;
+}
+
+if ($usage) {
+	echo 'call "' . $TOOLNAME . ' $task [$appName]' . PHP_EOL;
+	echo '$task: create-pot-files || convert-po-files || check-files' . PHP_EOL;
```

Comment:
Let's use symfony command style:

```suggestion
	echo 'Usage:' . PHP_EOL;
	echo ' ' . $TOOLNAME . ' <task> [<appName>]' . PHP_EOL;
	echo 'Arguments:' . PHP_EOL;
	echo ' task:            One of: create-pot-files, convert-po-files, check-files' . PHP_EOL;
```